resources/assets/js/annotations/annotatorContainer.vue

```diff
@@ -751,7 +754,9 @@ export default {
         Keyboard.on('C', this.selectLastAnnotation, 0, this.listenerSet);
     },
     mounted() {
+        // Event is used in laserpointer module
         Events.$emit('annotations.map.init', this.$refs.canvas.map);
+        this.map = this.$refs.canvas.map;
```

Comment:
The map must not become a reactive property (see previous review). While you can replace the other use of `Event` with props, the `map.init` event listener must stay.

Reply:
Just out of curiosity, why should the map not be made a reactive property?

Reply:
Vue will recursively walk through the whole object (which is very large) and make everything reactive that it can find. It will hook up the reactivity watchers/getters/setters to everything. This will make the whole OpenLayers system slow. With performance-critical paths/objects like this, you have to be very careful with Vue.